BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.h

```diff
@@ -47,12 +47,12 @@ typedef void (^BOXSessionTaskReplacedBlock)(NSURLSessionTask *oldSessionTask, NS
  * An API call is considered to have succeeded if it returns with an HTTP status code in the
  * 2xx range, with an exception made for 202. 202, 3xx, 4xx, and 5xx are all treated as errors.
  *
- * NSURLConnectionDataDelegate
+ * BOXURLSessionTaskDelegate
  * ===========================
- * BOXAPIOperation instances issue API calls using NSURLConnection. Each BOXAPIOperation acts
- * as the delegate for its own API call. [BOXAPIOperations](BOXAPIOperation) are not concurrent
- * NSOperations, so they keep the runloop they are running on open during the lifetime of the
- * request to enable receiving delegate callbacks.
+ * BOXAPIOperation instances issue API calls using NSURLSession. Each BOXAPIOperation acts
```

Comment:
I phrased it like this on purpose, saying we issue API calls using 'nsurlsession' (the general networking mechanism) and the operations are the delegates of their respective session task (see the next line).

What do you think? Would it be clearer to change it?